modules/integration/tests-common/ui-pages/src/main/java/org/wso2/emm/integration/ui/pages/platformConfiguration/PlatformConfiguration.java

```diff
@@ -24,8 +24,12 @@
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.wso2.emm.integration.ui.pages.UIElementMapper;
+
 import java.io.IOException;
 
+/**
+ * Class to represent the Platform Configuration Page.
```

Comment:
This comment doesn't seem to align with the pattern followed in other similar test cases.